HISTORY.md

```diff
@@ -1167,7 +1167,7 @@ This section lists changes that do not have deprecation warnings.
     to evaluate the wrapper. Consequently, package authors generally need to specialize
     `copy` and `copyto!` methods rather than `broadcast` and `broadcast!`. This also allows
     for more customization and control of fused broadcasts. See the
-    [Interfaces chapter](https://docs.julialang.org/en/latest/manual/interfaces/#man-interfaces-broadcasting-1)
+    [Interfaces chapter](https://docs.julialang.org/en/v1/manual/interfaces/#man-interfaces-broadcasting-1)
```

Comment:
I'm not sure if it makes sense to update the HISTORY file. Did you check that the updated links still make sense for the context?

Reply:
I think it would be ok. v1 would certainly be better than latest in HISTORY, I think.